lib/resources/r4/condition.yaml

```diff
@@ -167,14 +168,21 @@ fields:
     }
   binding:
     description: A category assigned to the condition.
```

Comment:
On the jira it says to update this to "A classification assigned......" is that what we want or did we change our minds to keep it as category?

Reply:
this description is for the field category, like in general. That description is for specifically the classification code. not sure where i would put it

Reply:
Ok fair enough then, not too worried about it